k8s/airflow-operator/README.md

```diff
@@ -0,0 +1,157 @@
+# Overview
```

Comment:
Can you revert the changes in `vendor`? Sync to master and check out the submodule at that version. You can run `make submodule/init` at the root level of this project.

Reply:
done